tests/test_s3/test_s3.py

```diff
@@ -1507,9 +1523,6 @@ def test_boto3_copy_object_with_versioning():
     obj1_version = client.get_object(Bucket='blah', Key='test1')['VersionId']
     obj2_version = client.get_object(Bucket='blah', Key='test2')['VersionId']
 
-    # Versions should be the same
-    obj1_version.should.equal(obj2_version)
```

Comment:
I'm not sure what the exact intent of the original author was here, but I don't think it's any relevant to the test case. The fact that this assertion was true was just a side-effect of the previous versionid implementation, but had no meaning AFAICT.

Reply:
I would assume it may have to do with differences between the `list` v1 and v2 APIs? Though I'm just guessing.

Reply:
Was this test supposed to test the `copy_object` API: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Client.copy_object ?

It's not clear what this test is trying to do.  Please feel free to fix it.

Reply:
> It's not clear what this test is trying to do. Please feel free to fix it.
> Was this test supposed to test the `copy_object` API

It tests that `copy_object` function actually updates the object version (see below). This is still true and I did not change this behavior. Apart from this specific assertion, the test case looks fine to me. I don't think there's something else to fix.